content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -398,248 +388,55 @@ to hold the configuration of the NGINX server:
-          app: sleep
-      template:
-        metadata:
-          labels:
-            app: sleep
-        spec:
-          containers:
-          - name: sleep
-            image: tutum/curl
-            command: ["/bin/sleep","infinity"]
-            imagePullPolicy: IfNotPresent
-            volumeMounts:
-            - name: nginx-client-certs
-              mountPath: /etc/nginx-client-certs
-              readOnly: true
-            - name: nginx-ca-certs
-              mountPath: /etc/nginx-ca-certs
-              readOnly: true
-          volumes:
-          - name: nginx-client-certs
-            secret:
-              secretName: nginx-client-certs
-          - name: nginx-ca-certs
-            secret:
-              secretName: nginx-ca-certs
-    EOF
-    {{< /text >}}
-
-1.  Define an environment variable to hold the name of the `sleep` pod:
-
-    {{< text bash >}}
-    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
-    {{< /text >}}
-
-1.  Use the deployed [sleep]({{< github_tree >}}/samples/sleep) pod to send requests to the NGINX server.
-    Since `nginx.example.com` does not actually exist and therefore DNS cannot resolve it, the following
-    `curl` command uses the `--resolve` option to resolve the hostname manually. The IP value passed in the
-    --resolve option (1.1.1.1 below) is not significant. Any value other than 127.0.0.1 can be used.
-    Normally, a DNS entry exists for the destination hostname and you would not use the `--resolve` option of `curl`.
-
-    {{< text bash >}}
-    $ kubectl exec "${SOURCE_POD}" -c sleep -- curl -v --resolve nginx.example.com:443:1.1.1.1 --cacert /etc/nginx-ca-certs/ca-chain.cert.pem --cert /etc/nginx-client-certs/tls.crt --key /etc/nginx-client-certs/tls.key https://nginx.example.com
-    ...
-    < HTTP/1.1 200 OK
-    ...
-    <!DOCTYPE html>
-    <html>
-    <head>
-    <title>Welcome to nginx!</title>
-    ...
-    {{< /text >}}
-
-1.  Verify that the server requires the client's certificate:
-
-    {{< text bash >}}
-    $ kubectl exec "$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -k --resolve nginx.example.com:443:1.1.1.1 https://nginx.example.com
-    ...
-    <html>
-    <head><title>400 No required SSL certificate was sent</title></head>
-    <body>
-    <center><h1>400 Bad Request</h1></center>
-    <center>No required SSL certificate was sent</center>
-    ...
-    </body>
-    </html>
-    {{< /text >}}
-
-### Redeploy the egress gateway with the client certificates
+### Mount client certificates to the egress gateway
 
 1. Create Kubernetes [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to hold the client's and CA
    certificates.
 
     {{< text bash >}}
-    $ kubectl create -n istio-system secret tls nginx-client-certs --key nginx.example.com/4_client/private/nginx.example.com.key.pem --cert nginx.example.com/4_client/certs/nginx.example.com.cert.pem
```

Comment:
But this way the readers will not know how to patch their deployment once they will need to mount client certificates for different sites. So maybe, instead of warning them, it is better just to show them how to patch the deployment?

Reply:
While I agree with how we should maybe add more information about patching, there is a new approach to this now. Which uses SDS with no file mounts required for certs and it's coming with 1.7 release. Instead of investing more time in this user doc which would eventually be deprecated in 1.9, we should focus on letting users migrate to SDS approach instead

Reply:
I mean to leave the example as it is now, with patching the deployment?

Reply:
No I mean leaving the example with this simplification + warning but I don't mind adding more context to the warning if need be

Reply:
Why this user doc will be deprecated in 1.9?

Reply:
There's plans to default to SDS instead of file-mounts for egress TLS. Same was done for ingress TLS with file-mount doc deprecated in 1.5 or 1.6

Reply:
> his user doc which would eventually be deprecated in 1.9
you probably meant the file-mounting aspect will be deprecated in 1.7, not the whole doc.

so don't you want to switch to SDS already in this task? to show SDS with client certificates per specific site?

Reply:
SDS is different task than this one for now. The doc will be deprecated in 1.9(hopefully), this approach is identical to the one taken for ingress SDS which worked out fine. This doc will be renamed with "file-mount" suffix. So the users can migrate to SDS over next two releases and then file-mount doc will be deprecated. The support for file-mounts will still exist but we'll not document it

Reply:
So I would continue showing deployment patching with file-mounting for this document, and write the doc with SDS, to perform migration as you describe.

Reply:
I've kept patching of gateway entirely, so I reverted those changes and I'll resolve this concern now